packages/backend/src/core/transaction-count/RpcTransactionUpdater.ts

```diff
@@ -52,6 +53,8 @@ export class RpcTransactionUpdater implements TransactionCounter {
     this.assessCount = opts?.assessCount ?? identity
     this.startBlock = opts?.startBlock ?? 0
     this.workQueueSizeLimit = opts?.workQueueSizeLimit ?? 200_000
+    const cache = new Cache(new InMemoryCacheBackend())
+    cache.wrapMethod(rpcClient, 'getBlockAtOrBefore')
```

Comment:
I am not a fan of using Cache here because it was generally unused and only kept around so that some scripts for optimism are still working and InMemoryCacheBackend was only created so that it it can be tested.

I think having an in memory map here would be more verbose but better. Also adding `// @deprecated` to cache would make this clear for anyone in the future